baseapp/baseapp.go

```diff
@@ -6,17 +6,16 @@ import (
 	"sort"
 	"strings"
 
-	dbm "github.com/cosmos/cosmos-db"
 	"github.com/cosmos/gogoproto/proto"
 	abci "github.com/tendermint/tendermint/abci/types"
 	"github.com/tendermint/tendermint/crypto/tmhash"
 	"github.com/tendermint/tendermint/libs/log"
 	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	dbm "github.com/tendermint/tm-db"
```

Comment:
curious why is this change needed?

Reply:
Fixed this, had to merge the latest from main